report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -47,76 +61,142 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
+If extreme values are unevenly distributed between the training and test sets, the evaluation of the model can be heavily skewed, leading to unreliable and misleading performance metrics.
+By redistributing extreme values evenly across the training folds, we ensure a more balanced and fair assessment of the model's capabilities during cross-validation.
+
+Secondly, this trade-off allows for a more stable and reliable assessment of the model's performance on typical data points.
+While the test set may be less representative of the full range of data, particularly regarding the rare extreme values, the evaluation focuses on the model's ability to generalize from the training data to new data within the same distribution range.
+This is prudent because, in many practical applications, the reliability of the model on typical data points is more critical than its performance on rare extremes.
+By using cross-validation in conjunction with a separate test set, we ensure that the model is robust and performs well under typical conditions.
+
+However, it is important to recognize the limitation this approach introduces: excluding extreme values from the test set means we can only confidently assess the model's performance within the range of the test set.
+If we receive predictions outside this range, we cannot reliably assert their accuracy, as the model has not been fully evaluated on such data.
+This could potentially make the model less useful in scenarios where predictions on extreme values are critical.
+Nevertheless, cross-validation partially mitigates this issue by allowing the model to learn from extreme values during the training and validation phases, thereby improving its robustness.
+Metrics obtained from cross-validation provide a broader picture of the model's accuracy and robustness across the entire dataset, including extreme values.
+
+Therefore, although our approach may make the test set less representative of the full dataset, it is a deliberate trade-off aimed at achieving a more accurate and reliable evaluation of the model's generalization performance.
 
 Our method is inspired by the approach described by \citet{andersonImprovedAccuracyQuantitative2017}.
 They employed a similar strategy to assess the performance of their PLS model, using k-fold cross-validation and a separate test set.
 Their process involved dividing the full set of laboratory data into five folds, using four for cross-validation and combining them as the final training set, while the fifth fold served as a test set.
-This approach ensured that each fold represented the full elemental compositional variation, and extreme targets were included in the training folds to handle a wider range of compositions.
-For consistency, we also use \(k=5\) for our cross-validation.
+For consistency, we also use $k = 5$ for our data partitioning.
+Given that the $k$-th fold is used as the test set, having $k=5$ results in 4 folds for cross-validation.
```

Comment:
You use a space elsewhere so probably should here too? 
Don't think it even affects the rendered PDF but just to make sure.
```suggestion
Given that the $k$-th fold is used as the test set, having $k = 5$ results in 4 folds for cross-validation.
```